traffic_ops/app/lib/API/Server.pm

```diff
@@ -400,40 +401,25 @@ sub get_servers_by_status {
 	my $orderby           = $self->param('orderby') || "hostName";
 	my $orderby_snakecase = lcfirst( decamelize($orderby) );
 
+	my $forbidden;
```

Comment:
not sure why this method needed to change (get_servers_by_status) as it seems unrelated to delivery services.